concordium-consensus/src/Concordium/External.hs

```diff
@@ -891,14 +891,17 @@ decodeBlockHash blockcstr = readMaybe <$> peekCString blockcstr
 decodeAccountAddress :: CString -> IO (Either String AccountAddress)
 decodeAccountAddress acctstr = addressFromBytes <$> BS.packCString acctstr
 
--- |Decode a null-terminated string as either an account address (base-58) or a
+-- |Decode a null-terminated string as either an account address (base-58), account index (Int) or a
```

Comment:
Account index is not an `Int`. There is a type `AccountIndex` defined in concordium-base (and it is defined as a wrapper around Word64).

Reply:
You are right. I actually defined it using `AccountIndex` in the `AccountIdentifier`.